sdk/trace/batch_span_processor.go

```diff
@@ -120,6 +139,55 @@ func NewBatchSpanProcessor(exporter SpanExporter, options ...BatchSpanProcessorO
 	return bsp
 }
 
+var processorIDCounter atomic.Int64
+
+// nextprocessorIDCounter returns an identifier for this batch span processor,
+// starting with 0 and incrementing by 1 each time it is called.
+func nextprocessorIDCounter() int64 {
+	return processorIDCounter.Add(1) - 1
+}
```

Comment:
nit like hell :wink: 

```suggestion
// nextProcessorIDCounter returns an identifier for this batch span processor,
// starting with 0 and incrementing by 1 each time it is called.
func nextProcessorIDCounter() int64 {
	return processorIDCounter.Add(1) - 1
}
```

Reply:
done